src/pages/admin/creators/edit.tsx

```diff
@@ -115,10 +124,10 @@ const AdminEditCreators = ({ marketplace }: AdminEditCreatorsProps) => {
       subdomain: marketplace.subdomain,
       name: marketplace.name,
       description: marketplace.description,
-      creators: marketplace.creators.map(({ creatorAddress }) => ({
+      creators: marketplace.creators?.map(({ creatorAddress }) => ({
         address: creatorAddress,
       })),
-      transactionFee: marketplace.auctionHouse.sellerFeeBasisPoints,
+      transactionFee: marketplace.auctionHouses[0].sellerFeeBasisPoints,
```

Comment:
The user should probably be able to pick different transaction fee based on spl token. Thoughts @damianr 

Which means the spl UI should support showing input for transaction fee.